src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -1066,8 +1066,6 @@ void setOptimalIndex(int newOptimalIdx)
          */
         void rtcpTransform(ByteArrayBuffer baf)
         {
-            SeenFrame maxSentFrame = this.maxSentFrame;
-
```

Comment:
after talking with @gpolitis it is only this method which needs to have `maxSentFrame` protected, as it can be called from a different thread than `accept`.  it's only written in `accept` itself (and `setTL0Idx`, which is called from `accept`) so the write there are fine.